activerecord/test/models/user.rb

```diff
@@ -5,7 +5,7 @@ class User < ActiveRecord::Base
   has_secure_token :auth_token
 
   has_and_belongs_to_many :jobs_pool,
-    class_name: Job,
+    class_name: "Job",
```

Comment:
@rafaelfranca in Slack you mentioned that "in fact passing `class_name` as a constant doesn’t work at all" but we've been using it here